vspace/src/main/java/edu/asu/diging/vspace/web/publicview/ExhibitionSequencesController.java

```diff
@@ -65,7 +65,9 @@ public String sequence(Model model, @PathVariable("sequenceId") String sequenceI
 
         List<ISlide> slides = sequenceManager.getSequence(sequenceId).getSlides();
         if (slides.size() == 0) {
-            throw new SlidesInSequenceNotFoundException();
+            model.addAttribute("showAlert", true);
+            model.addAttribute("message", "Sorry, slides haven't been added to this module yet.");
```

Comment:
message should say "Sorry, module has not been configured yet."

Reply:
Addressed this and have changed the message to appropriate template.